contents/handbook/growth/sales/new-sales.md

```diff
@@ -0,0 +1,141 @@
+
+You should then follow up with a standard email/Slack message that:
+
+- Summarizes what they’re after and how Posthog is a great/bad fit
+- Lays out next steps on both sides
+- Shares a proposed timetable for the evaluation and onboarding process
+- Includes any useful lin=ks (e.g. Docs page, competitor comparisons, relevant case studies)
+
+The AE acts as the support person during this period, usually answering questions in Slack. 
+
+We usually set up the following trials depending on likely contract size:
+
+- $20-60k - 2 weeks
+- $60k+ - 1 month
+
+Depending on the product they are most interested in, we usually set up the customer set up the following:
+
+- Product/web analytics + session replay: get tracking set up, turn on replay, privacy controls, figure out user ID, get set up insights/dashboards.
+- Feature flags + experiments: snippet, FF in the code, person ID and properties for targeting, deploy flag, run the experiment. 
+- Surveys: deploy a survey
+- Data warehouse: TBD
+
+Set yourself a reminder for halfway through their evaluation period to look at their usage then put together a commercial proposal. 
+
+**5. Security review**
+
+Most customers don't need this beyond sharing our existing documentation. This step often occurs in parallel with product evaluation.  
+
+If you need help with anything data privacy or [MSA-related](/handbook/growth/sales/contracts), ping Fraser for help. 
+
+**6. Commercial evaluation**
+
+The [Contracts](/handbook/growth/sales/contracts) page has full guidance on the nuts and bolts of how to put together a commercial proposal. 
+
+By this point, you may have started running into some objections. These are the most common, and how to handle:
+
+- Gap in the product - introduce the customer to the relevant product engineer to build together (but first agree with product team if it’s a reasonable ask). We have found this approach works exceptionally well for our newer products. 
+- Pricing issue - understand their budget, just make sure that the deal overall isn't margin negative. You can also help them tune their usage to lower costs. We don't buy customers out of existing contracts. 
+- Performance (e.g. slow dashboards) - for Extra Large, usually get Tim involved, or he can loop in the right engineer to help. 
+- Confidence in PostHog - often [this Handbook page](/handbook/finance) is enough. For Very Large companies who need to be sold a bit more on the company vision, you can get James H involved.
+
+**7. Closed - won**
+
+Hooray! This is defined as when the contract is signed by _everyone_. 'They're about to sign' - NOT CLOSED. 'I've sent a DocuSign' - NOT CLOSED EITHER.  
+
+Once the contract is signed, add it to the [signed contracts folder](https://drive.google.com/drive/folders/1ccWPIrY9dXwrzJwppFI2pm3kFIcOb8j8) and get them set up with [billing](/handbook/growth/sales/billing). 
```

Comment:
I don't think we need to do this - they are all stored in PandaDoc and will also be linked to the HubSpot deal object

Reply:
Updated!